src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs

```diff
@@ -53,11 +53,11 @@ public static void SendingResult(ILogger logger, string? invocationId, ObjectMet
     [LoggerMessage(9, LogLevel.Trace, "'{HubName}' hub method '{HubMethod}' is bound.", EventName = "HubMethodBound")]
     public static partial void HubMethodBound(ILogger logger, string hubName, string hubMethod);
 
-    [LoggerMessage(10, LogLevel.Debug, "Canceling stream for invocation {InvocationId}.", EventName = "CancelStream")]
-    public static partial void CancelStream(ILogger logger, string invocationId);
+    [LoggerMessage(10, LogLevel.Debug, "Canceling invocation {InvocationId}.", EventName = "CancelInvocation")]
```

Comment:
We can't change existing logs' EventName or Id. So let's make a new log for the more generic "CancelInvocation" log. Also might want to mention "Canceling invocation or stream {InvocationId}."